constraint/pkg/client/client.go

```diff
@@ -157,34 +158,71 @@ func createTemplatePath(target, name string) string {
 	return fmt.Sprintf(`templates["%s"]["%s"]`, target, name)
 }
 
-// CreateCRD creates a CRD from template
-func (c *Client) CreateCRD(ctx context.Context, templ *templates.ConstraintTemplate) (*apiextensions.CustomResourceDefinition, error) {
+// templateLibPrefix returns the new lib prefix for the libs that are specified in the CT.
+func templateLibPrefix(target, name string) string {
+	return fmt.Sprintf("libs.%s.%s", target, name)
+}
+
+// validateTargets handles validating the targets section of the CT.
+func (c *Client) validateTargets(templ *templates.ConstraintTemplate) (*templates.Target, TargetHandler, error) {
 	if err := validateTargets(templ); err != nil {
-		return nil, err
+		return nil, nil, err
 	}
+
+	if len(templ.Spec.Targets) != 1 {
+		panic("consistency error, expected exactly 1 item in targets")
```

Comment:
We probably don't want to panic, as that would cause one bad template to make the controller/webhook crash loop.

Reply:
now returns an error